azurelinuxagent/common/protocol/wire.py

```diff
@@ -1059,7 +1061,12 @@ def send_encoded_event(self, provider_id, event_str, encoding='utf8'):
             header = self.get_header_for_xml_content()
             # NOTE: The call to wireserver requests utf-8 encoding in the headers, but the body should not
             #       be encoded: some nodes in the telemetry pipeline do not support utf-8 encoding.
-            resp = self.call_wireserver(restutil.http_post, uri, data, header)
+
+            # if it's important event flush, we use less throttle delay(to avoid long delay to complete this operation)) on throttling errors
+            if flush:
+                resp = self.call_wireserver(restutil.http_post, uri, data, header, max_retry=3, telemetry_throttle_delay=TELEMETRY_FLUSH_THROTTLE_DELAY_IN_SECONDS)
+            else:
+                resp = self.call_wireserver(restutil.http_post, uri, data, header, max_retry=3, telemetry_throttle_delay=TELEMETRY_THROTTLE_DELAY_IN_SECONDS)
```

Comment:
Default retry was previously 6, we're allowing less retries now?

Reply:
My intention for lower number is higher retires contribute to reach throttle limit very frequently. Do you think default 6 would help on any general errors?